packs/heritages/goblin/irongut-goblin.json

```diff
@@ -18,37 +18,54 @@
             "title": "Pathfinder Player Core"
         },
         "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Goblin.IrongutGoblin.BonusToggle",
+                "option": "irongut-goblin-ingested",
+                "toggleable": true
+            },
             {
                 "key": "FlatModifier",
-                "label": "Irongut Goblin (vs. afflictions & sickened)",
                 "predicate": [
-                    "ingested",
-                    {
-                        "or": [
-                            "affliction",
-                            "inflicts:sickened"
-                        ]
-                    }
+                    "irongut-goblin-ingested"
```

Comment:
I would rather stick with the above but made more proper, with `item:type:affliction` and `item:type:ingested`. I generally think these kind of rarer things don't need toggles, and can just be flipped on in the dialog until we can properly automate them.